ddtrace/monkey.py

```diff
@@ -200,7 +201,15 @@ def patch(raise_errors=True, **patch_modules):
     )
 
 
+@deprecated(
+    message="This function will be removed.",
+    version="1.0.0",
+)
 def patch_module(module, raise_errors=True):
+    return patch_module(module, raise_errors=raise_errors)
```

Comment:
```suggestion
    return _patch_module(module, raise_errors=raise_errors)
```

otherwise this will infinitely call itself